kettle/troubleshoot.md

```diff
@@ -0,0 +1,39 @@
+## Troubleshooting Kettle data collection issue
```

Comment:
Maybe this page shouldn't be about troubleshooting at all, but just be instructions on how to restart Kettle. The readme can refer to the troubleshooting.